cm_brainClasses.py

```diff
@@ -314,15 +305,14 @@ def reset(self):
         self.outvars = {"rx": 0, "ry": 0, "rz": 0,
                         "px": 0, "py": 0, "pz": 0}
         self.tags = self.sim.agents[self.userid].access["tags"]
-        # self.tags = {}
         self.agvars = self.sim.agents[self.userid].agvars
 
     def execute(self):
         """Called for each time the agents needs to evaluate"""
         actv = bpy.context.active_object
         self.isActiveSelection = actv is not None and actv.name == self.userid
         self.reset()
-        self.randstate = hash(self.userid) + self.sim.framelast
+        self.randstate = hash(self.userid) + self.sim.framelast  # randstate should be defined in __init__!
```

Comment:
Actually this isn't used anywhere else in the project so you could just make it a local variable.